packages/code-infra/src/cli/cmdPublishNewPackage.mjs

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env node
+
+import fs from 'node:fs/promises';
+import os from 'node:os';
+import path from 'node:path';
+import { $ } from 'execa';
+
+/**
+ * @typedef {Object} Args
+ * @property {string[]} pkg Package names to create
+ * @property {boolean} [dryRun] If true, will only log the commands without executing them
+ * @property {string} [tag] NPM dist tag to publish to
+ */
+
+export default /** @type {import('yargs').CommandModule<{}, Args>} */ ({
+  command: 'publish-new-package [pkg...]',
+  describe: 'Publish new empty package(s) to the npm registry.',
+  builder: (yargs) =>
+    yargs
+      .positional('pkg', {
```

Comment:
Since we already have a function that can detect unpublished packages that we use in the `publish` command, how about we just use it here as well and avoid the need for this parameter?